### PR TITLE
Pull Request for release 0.9.0beta1

### PR DESCRIPTION
**Pull request by notoraptor**
This is a PR to prepare release 0.9.0beta1. It will be updated to match theano/master branch updates until we release.

What do you think ? @nouiz @lamblin

### CODE REVIEW

File: doc/library/config.txt [hunk 1]
```diff
@@ -585,7 +585,7 @@ import theano and print the config variable, as in:
 .. attribute:: config.warn.ignore_bug_before
 
     String value: ``'None'``, ``'all'``, ``'0.3'``, ``'0.4'``, ``'0.4.1'``, ``'0.5'``,
-    ``'0.6'``, ``'0.7'``, ``'0.8'``, ``'0.8.1'``, ``'0.8.2'``
+    ``'0.6'``, ``'0.7'``, ``'0.8'``, ``'0.8.1'``, ``'0.8.2'``, ``'0.9.0beta1'``
```
Comment (lamblin) [inline doc/library/config.txt]: I think you should just put 0.9 in there, we will not try to distinguish between beta / rc here.

File: NEWS.txt [hunk 2]
```diff
@@ -3,282 +3,204 @@ Release Notes
...
+ - Support newer Mac and Windows version
+ - Conda packages for Mac, Linux and Windows
+ - Theano scripts now works on Windows
+ - scan with checkpoint (trade off between speed and memory usage, useful for long sequences)
+ - Added a bool dtype
+ - Added a deprecation message for the old backend.
+
+ - New back-end:
+   - float16 storage
+   - print pci bus id
```
Comment (notoraptor) [inline NEWS.txt]: I suggest: `better mapping between theano device number and nvidia-smi number, using the PCI bus ID of graphic cards`

File: NEWS.txt [hunk 3]
```diff
@@ -3,282 +3,204 @@ Release Notes
...
+   - More pooling support on GPU when cudnn isn't there.
```
Comment (notoraptor) [inline NEWS.txt]: That is, I should add "use cuDNN" to this list ?

**Comment by nouiz** [comment]
"Continue to use CuDNN" with RNN support.

**Comment by notoraptor** [comment]
@nouiz in "GPU" news section, there is that line: `CuDNN: support versoin 5.1 and wrap batch normalization (2d and 3d) and RNN functions`

May it be what you want ?

File: NEWS.txt (hunk 3 above)
Comment (lamblin) [inline NEWS.txt]: I think @nouiz means that we should use the spelling "cuDNN" instead of "cudnn"

File: doc/library/config.txt (hunk 1 above)
Comment (lamblin) [inline doc/library/config.txt]: Actually, this has already been done in #5437, so if you remove the change, it may make the merge easier.

**Comment by notoraptor** [comment]
I have rebased + updated the PR.

**Comment by notoraptor** [comment]
Last PRs reviewed. I have also rebased the branch. The number and the order of contributors have not changed.

About new PRs, I'm not sure if I should add new lines to NEWS for the following:
* https://github.com/Theano/Theano/pull/4659 (implementing stack trace copy over and tests for various ops)
* https://github.com/Theano/Theano/pull/5343 (Broadcasting tutorial)
* https://github.com/Theano/Theano/pull/5421 (Cusolver using Cholesky decomposition for symmetric matrices

**Comment by nouiz** [comment]
For the first 2, no need to updates news. For the last, it was already there. No need to add stuff there.

**Pull request merged**